controlplane/kubeadm/config/crd/bases/controlplane.cluster.x-k8s.io_kubeadmcontrolplanes.yaml

```diff
@@ -10,13 +10,32 @@ metadata:
 spec:
   additionalPrinterColumns:
   - JSONPath: .status.ready
-    description: Total number of clusters ready
+    description: KubeadmControlPlane API Server is ready to receive requests
     name: Ready
     type: boolean
   - JSONPath: .status.initialized
-    description: Cluster is in initialized state or not(True/False)
+    description: This denotes whether or not the control plane has the uploaded kubeadm-config
+      configmap
     name: Initialized
     type: boolean
+  - JSONPath: .status.replicas
+    description: Total number of non-terminated machines targeted by this control
+      plane
```

Comment:
This looks like it should be on the preceding line, is this a result of an invisible embedded character in the kubebuilder annotation description or line length? Or is it an accidental manual modification?

Reply:
It's not manual. Let me check in the annotation description